catalystwan/__init__.py

```diff
@@ -71,7 +71,7 @@ def list_package_sources() -> List[Path]:
 pkg_src_list = list_package_sources()
 
 
-if environ.get("catalystwan_DEVEL") is not None:
```

Comment:
Not sure what the environment variable naming conventions are.
Let's stick with snake case for our library not mix it with CAPS.
`catalystwan_{lower_case_suffix}`

Reply:
will be resolved in next PR